README.adoc

```diff
@@ -1,3 +1,10 @@
+= SonarCloud Analysis with GitHub Actions
```

Comment:
What about adding a link to https://sonarcloud.io/documentation/getting-started/github/ ?
In case the user finds this project example from elsewhere, and did not configure correctly his github account etc...

Reply:
I added it to the documentation links